app/editor/src/features/admin/ingests/ContentReferenceList.tsx

```diff
@@ -31,23 +34,35 @@ export interface IContentReferenceListProps {}
 
 const ContentReferenceList: React.FC<IContentReferenceListProps> = (props) => {
   const [{ requests }] = useApp();
-  const [, api] = useContentReferences();
+  const [, { findContentReferences, findContentIds }] = useContentReferences();
   const { values } = useFormikContext<IIngestModel>();
-
-  const sources = [
-    values.source?.code,
-    ...(values.configuration?.sources?.split('&').map((s: string) => s.split('=').slice(-1)) ?? []),
-  ].filter((s) => s !== undefined) as string[];
+  const [{ sources }] = useLookup();
 
   const [filter, setFilter] = React.useState<IContentReferenceListFilter>({
     ...defaultContentReferenceFilter,
-    sources: sources,
+    sources: values.source?.code ? [values.source.code] : [],
     mediaTypeIds: [values.mediaTypeId],
   });
   const [page, setPage] = React.useState<IPage<IContentReferenceModel>>(
     defaultContentReferencePage,
   );
 
+  // Extract possible sources for this ingest from the configuration, or list all options.
+  const configSources = [values.source?.code]
```

Comment:
In Openshift this tab doesn't even work because of performance issues.  Hopefully this helps.